modules/openapi-generator/src/main/resources/lumen/composer.mustache

```diff
@@ -19,14 +19,38 @@
             "homepage": "https://openapi-generator.tech"
         }
     ],
+    "type": "project",
     "require": {
-        "php": ">=5.5.9",
-        "laravel/lumen-framework": "5.2.*",
+        "php": ">=7.1.3",
```

Comment:
Given that this is a breaking change, I would suggest incorporate this change after the 1st stable release (3.0.0, which is due by the end of this month).

Reply:
agree. php 7.1 still not widely installed. Don't know why Lumen jumped from php5.5 to 7.1 in such aggressive way. But newly created lumen projects preventing developers from using php7.0- at least on my side. btw: Lumen 5.2 is quite obsolete. And per #7530(swagger-api/swagger-codegen#7530) in swagger-codegen, currently no direct support for Lumen 5.3+ in both swagger and this tool. Am I the only one using both Lumen and swagger/openapi-gen?

Reply:
Lumen 5.2     PHP >= 5.5.9
Lumen 5.3     PHP >= 5.6.4
Lumen 5.4     PHP >= 5.6.4
Lumen 5.5     PHP >= 7.0
Lumen 5.6     PHP >= 7.1.3